README.md

```diff
@@ -52,14 +53,32 @@ vagrant ssh [hostname]  # ssh into the given hostname (specified at the top of t
 vagrant provision       # rerun the ansible provisioning steps on the running machines
 ```
 
-## Running the demo
+## Running the demo with HDP
```

Comment:
Not necessarily, if they want to include HDP in the data flow, then they need to get HDP started before anything relevant will be in the webapp